releasenotes/v1.9.3.md

```diff
@@ -0,0 +1,21 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v1.9.2
+
+## :bug: Bug Fixes
+
+- bump golang to 1.23.6 (#2322)
+
+## :seedling: Others
+
+- Bump IPAM to 1.9.4 (#2346)
+- Bump CAPI to 1.9.5 (#2344)
+- Bump sigs.k8s.io/controller-runtime from 0.19.5 to 0.19.6 (#2330)
+- Bump the kubernetes group across 5 directories with 7 updates (#2327)
```

Comment:
Can you update this to match the current PR title? For some reason, our automation takes the original title here, and not the current one.